docs/src/modules/sandbox/CodeSandbox.ts

```diff
@@ -35,7 +35,29 @@ function openSandbox({ files, codeVariant, initialFile }: any) {
 
 function createReactApp(demoData: DemoData) {
   const ext = getFileExtension(demoData.codeVariant);
-  const { title, githubLocation: description } = demoData;
+  const { title, relativeModules = [], githubLocation: description } = demoData;
+
+  if (relativeModules) {
+    relativeModules.forEach(({ module, raw: content }) => {
+      // remove exports from relative module
+      content = content.replace(/export( )*(default)*( )*\w+;|export default|export/gm, '');
+      // replace import statement with relative module content
+      // the module might be imported with or without extension, so we need to cover all cases
+      // E.g.: /import .* from '(.\/top100Films.js|.\/top100Films)';/
+      const extensions = ['', '.js', '.jsx', '.ts', '.tsx', '.css', '.json'];
+      const patterns = extensions
+        .map((ex) => {
+          if (module.endsWith(ex)) {
+            return module.replace(ex, '');
+          }
+          return '';
+        })
+        .filter(Boolean)
+        .join('|');
+      const importPattern = new RegExp(`import .* from '(${patterns})';`);
+      demoData.raw = demoData.raw.replace(importPattern, content);
+    });
+  }
 
   const files: Record<string, object> = {
```

Comment:
Actually, why don't we just include the additional files here? That way we won't need to do any magic in copying the content in the same file.

Reply:
Let me explore that possibility and report

Reply:
Works well! I've made the change